x-pack/plugins/cloud/server/plugin.ts

```diff
@@ -40,6 +40,14 @@ export interface CloudSetup {
    * The full URL to the Kibana deployment.
    */
   kibanaUrl?: string;
+  /**
+   * The full URL to the serverless projects.
+   */
+  projectsUrl?: string;
```

Comment:
Looking at this naming I'm not convinced it is 100% correct as we default it to a path, not a URL in our serverless config. CC @Dosant